src/smc_sagews/smc_sagews/tests/README.md

```diff
@@ -0,0 +1,25 @@
+# Pytest suite for sage_server
+
+## Purpose of These Tests
+
+- verify that sage worksheets return correct results for given cell inputs
+- back-end only test, uses message protocol to run tests against running sage_server
+- NOT to provide unit tests of functions internal to sagews modules
+
+## How to Use These Tests
+
+### Prerequisites
+
+- pytest must be installed
+- make sure ~/.smc/sage_server/sage_server.log exists and has the
+   current port number around line 3, like this:
+   ```
+   9240 (2016-09-07 04:03:39.455): Sage server 127.0.0.1:43359
+   ```
```

Comment:
tied to my comment in the fixture about error messages, maybe it's better to explain here, that the sage server needs to be running and that there is this logfile? i.e. just the fact that the logfile exists is not enough, technically :-)